test/account_multisig.test.js

```diff
@@ -82,6 +82,17 @@ describe('deployMultisig key rotations', () => {
 
 describe('account2fa transactions', () => {
 
+    test('add app key before deployMultisig', async() => {
+        let account = await testUtils.createAccount(nearjs);
+        const appPublicKey = KeyPair.fromRandom('ed25519').getPublicKey();
+        const appMethodNames = ['some_app_stuff','some_more_app_stuff'];
+        await account.addKey(appPublicKey.toString(), 'foobar', appMethodNames.join(), new BN(parseNearAmount('0.25')));
+        account = await getAccount2FA(account);
+        const keys = await account.getAccessKeys();
+        expect(keys.find(({ public_key }) => appPublicKey.toString() === public_key)
+            .access_key.permission.FunctionCall.method_names.toString()).toEqual(appMethodNames.toString());
```

Comment:
should actually check arrays are equal instead of checking `.toString()` is equal, which I assume is passing the test disregarding wrong input (`appMethodNames.join()`).

@mattlockyer I guess it is needed to merge https://github.com/near/near-api-js/pull/436/files for multiple methods support?

Reply:
Yes